cms/templates/cms/home_page.html

```diff
@@ -44,97 +45,134 @@
-                  <div class="info-wrap">
-                    <div class="title">{{ program }}</div>
-                    <div class="description">
-                      {% if program.description %}
-                      {{ program.description }}
-                      {% else %}
-                      No description available for this program.
-                      {% endif %}
-                    </div>
-                    <div class="media-footer">
-                      {% if program.course_set.count == 1 %}
-                      <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} course</span></div>
-                      {% else %}
-                      <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} courses</span></div>
+                        <img src="{% static 'images/course-thumbnail.png' %}"
+                             alt="course image for {{ program }}" width="345" height="265"/>
                       {% endif %}
+                      <div class="program-title">{{ program }}</div>
+                      <div class="num-courses">
+                        {% if program.course_set.count == 1 %}
+                          <span class="meta-courses">{{ program.course_set.count }} course</span>
+                        {% else %}
+                          <span class="meta-courses">{{ program.course_set.count }} courses</span>
+                        {% endif %}
+                      </div>
                     </div>
-                  </div>
-                </div>
+
                 {% if program.programpage %}
                   </a>
                 {% endif %}
-              </li>
-              {% endfor %}
-              <li class="col-sm-6">
-                <div class="media-item placeholder-card">
-                  <br/><br/><br/>
-                  <div class="container">
-                    <div class="col-sm-12 title-coming-soon text-center">
-                      MIT has plans to bring more MicroMasters programs to motivated learners like you.
-                      Please sign up for our <a href="#" data-toggle="modal"
-                      data-target="#subscribeAlert">mailing list</a>
-                      to hear about new MicroMasters programs.
-                      Also, don't hesitate to let us know about what MicroMasters you’d like to see.
-                    </div>
-                    <div class="row">
-                      <div class="col-sm-12">
-                        <button class="btn btn-danger button-subscribe-alerts center-block"  data-toggle="modal"
-                       data-target="#subscribeAlert">Yes, tell me when MicroMasters are available</button>
-                      </div>
-                    </div>
-                  </div>
-               </div>
-              </li>
-            </ul>
+                <div class="program-thumbnail-description">
+                  {% if program.description %}
```

Comment:
I changed this to only show Program description for now, since ProgramPage description is long text that we do not want to use for the thumbnail.